lib/stanford-mods/geo_spatial.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: UTF-8
-require 'logger'
+# require 'logger'
```

Comment:
unclear, there are some inline comments a couple places about adding logging, and potentially we will some some debug logging for geo.  probably never an actual issue though, since this file is reopening the Record class.  i can remove it if you like.